CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+Fix config smtp auth user and password: unbind them from config smtp secure flag
```

Comment:
No related issue?

Reply:
fixed in https://github.com/telefonicaid/perseo-fe/pull/516/commits/317b1b3b9558956013609db45233d3f77ed1fe7e